samples/sysbuild/with_mcuboot/boards/stm32h750b_dk.conf

```diff
@@ -0,0 +1 @@
+CONFIG_STM32_MEMMAP=y
```

Comment:
this file should be submitted to https://github.com/mcu-tools/mcuboot otherwise no other application will work

Reply:
if this is needed when e.g. flash is enabled, should just do it in the board `Kconfig.defconfig` file

Reply:
what conditions is it needed in? If a user takes hello world and enables MCUboot on it using sysbuild, will it work? If not, that is a problem